static/js/redirection.js

```diff
@@ -79,3 +79,28 @@ window.addEventListener("load", () => {
     navigator.serviceWorker.register(serviceWorkerPath);
   }
 });
+
+const kGoToNextToken = 'gimmenext';
+let keyPressState = 0;
+document.addEventListener('keypress', (e) => {
+  if (kGoToNextToken[keyPressState] === e.key) {
+    keyPressState++;
+    if (keyPressState === kGoToNextToken.length) {
+      gotoNext();
+      keyPressState = 0;
+    }
+  } else {
+    keyPressState = 0;
+  }
+
+  function gotoNext() {
+    let newPath = '/docs/next/intro';
+    if (window.location.pathname.startsWith('/python'))
+      newPath = '/python/docs/next/intro';
+    if (window.location.pathname.startsWith('/java'))
+      newPath = '/java/docs/next/intro';
+    if (window.location.pathname.startsWith('/dotnet'))
+      newPath = '/dotnet/docs/next/intro';
+    window.location.href = newPath;
+  }
+});
```

Comment:
Why does this only take you to the intro instead of preserving the current page?

Reply:
in general good idea but probably not worth implementing for now.